LifeCycleAssessment_Engine/Compute/EvaluateEnvironmentalProductDeclarationByVolume.cs

```diff
@@ -47,11 +48,11 @@ public static partial class Compute
         [Input("field", "Filter the provided EnvironmentalProductDeclaration by selecting one of the provided metrics for calculation.")]
         [Input("exactMatch", "If true, the evaluation method will force an exact LCA phase match to solve for.")]
         [Output("quantity", "The total quantity of the desired metric based on the EnvironmentalProductDeclarationField.")]
-        private static EnvironmentalMetricResult EvaluateEnvironmentalProductDeclarationByVolume(IElementM elementM, List<LifeCycleAssessmentPhases> phases, EnvironmentalProductDeclarationField field = EnvironmentalProductDeclarationField.GlobalWarmingPotential, bool exactMatch = false)
+        private static EnvironmentalMetricResult EvaluateEnvironmentalProductDeclarationByVolume(IElementM elementM, List<LifeCycleAssessmentPhases> phases, MaterialComposition materialComposition, EnvironmentalProductDeclarationField field = EnvironmentalProductDeclarationField.GlobalWarmingPotential, bool exactMatch = false)
         {
             List<double> epdVal = elementM.GetEvaluationValue(field, phases, QuantityType.Volume, exactMatch);
```

Comment:
This should be updated to call to the new GetEvaluationValue method with MaterialComposition included to eliminate the redundant MaterialComposition call within the currently called method.